modules/network_synchronizer/data_buffer.cpp

```diff
@@ -135,14 +136,14 @@ void DataBuffer::dry() {
 }
 
 void DataBuffer::seek(int p_bits) {
-	ERR_FAIL_INDEX(p_bits, metadata_size + bit_size);
+	CRASH_BAD_INDEX(p_bits, metadata_size + bit_size);
 	bit_offset = p_bits;
 }
 
 void DataBuffer::shrink_to(int p_metadata_bit_size, int p_bit_size) {
-	ERR_FAIL_COND_MSG(p_metadata_bit_size < 0, "Metadata size can't be negative");
-	ERR_FAIL_COND_MSG(p_bit_size < 0, "Bit size can't be negative");
-	ERR_FAIL_COND_MSG(buffer.size_in_bits() < (p_metadata_bit_size + p_bit_size), "The buffer is smaller than the new given size.");
+	CRASH_COND_MSG(p_metadata_bit_size < 0, "Metadata size can't be negative");
+	CRASH_COND_MSG(p_bit_size < 0, "Bit size can't be negative");
+	CRASH_COND_MSG(buffer.size_in_bits() < (p_metadata_bit_size + p_bit_size), "The buffer is smaller than the new given size.");
```

Comment:
I would leave this as ERR_COND. This is not in direct control of the user, but if someone will use it, I think it's better to warn it, instead to crash the program, if he is doing some mistakes.